draft-ietf-quic-multipath.md

```diff
@@ -379,25 +379,26 @@ should not cause linkability issue.
 
 # Path Management {#path-management}
 
-After completing the handshake, endpoints have agreed to enable
-multipath support. They can start using multiple paths when both endpoints
-have issued available connection IDs for at least one unused Path ID.
-This document does not discuss when a client decides to initiate a new path.
+After completing the handshake indicating
+multipath support, endpoints can start using multiple paths when both endpoints
+have issued available connection IDs for at least one unused, common Path ID,
+as the same Path ID is used in both directions.
+
+This documents specfies path initiation (see Section {{path-initiation}}),
+issuing and retirement of per-path connection IDs (see Section
+{{consume-retire-cid}}), path status management (see Section {{path-state}}) and
+path closure (see Section {{path-close}}).
+However, this document does not specify when a client decides to initiate or close a path,
+or how multiple active paths are used for packet sending.
 
 ## Path Initiation and Validation {#path-initiation}
 
-To open a new path, an endpoint MUST use a connection ID (see {{Section 9.5 of QUIC-TRANSPORT}}) associated with
-a new, unused Path ID  (see {{explicit-path-id}}).
-Instead of NEW_CONNECTION_ID frame as specified in {{Section 19.15 of QUIC-TRANSPORT}},
-each endpoint uses the PATH_NEW_CONNECTION_ID frame as specified in this extension
-to issue Path ID-specific connections IDs.
-The same Path ID is used in both directions. As such to open
-a new path, both sides need at least
-one connection ID (see {{Section 5.1.1 of QUIC-TRANSPORT}}), which is associated
-with the same, unused Path ID. When the peer receives the PATH_CHALLENGE,
-it MUST pick a Connection ID with the same Path ID for sending the PATH_RESPONSE.
-
-When the multipath extension is negotiated, a client that wants to use an
+To open a new path, an endpoint MUST use a new connection ID associated
+with unused Path ID.
```

Comment:
```suggestion
with an unused Path ID.
```

Reply:
Please also fix the upper case "T" in "To open"